site/en/docs/extensions/reference/declarativeNetRequest/index.md

```diff
@@ -281,140 +322,90 @@ When rules are applied to browsers with pages in the service worker's cached sto
+
+`*://*.example.com/*`
+
+The following example shows how to redirect a request from example.com to a page within the extension itself. The extension path `/a.jpg` resolves to `chrome-extension://EXTENSION_ID/a.jpg`, where `EXTENSION_ID` is the ID of your extension.
+
+```json
+{
+  "id": 1,
+  "priority": 1,
+  "action": { "type": "redirect", "redirect": { "extensionPath": "/a.jpg" } },
+  "condition": {
+    "urlFilter": "https://www.example.com",
+    "resourceTypes": ["main_frame"]
+  }
+}
+```
+
+The following uses the `"transform"` key to redirect to a subdomain of example.com. It uses a domain name anchor ("||") to intercept requests with any scheme from example.com. The `"scheme"` key in `"transform"` specifies that redirects to the subdomain will always use "https".
+
+```json
+{
+  "id": 1,
+  "priority": 1,
+  "action": {
+    "type": "redirect",
+    "redirect": {
+      "transform": { "scheme": "https", "host": "new.example.com" }
     }
   },
-  {
-    "id" : 9,
-    "priority": 1,
-    "action" : {
-      "type" : "block"
-    },
-    "condition" : {
-      "urlFilter" : "script.js",
-      "resourceTypes" : ["script"]
+  "condition": {
+    "urlFilter": "||example.com",
+    "resourceTypes": ["main_frame"]
+  }
+}
+```
+
+The following example uses regular expressions to redirect from `https://www.abc.xyz.com/path` to `https://abc.xyz.com/path`. In the `"regexFilter"` key notice how periods are escaped and that the capturing group selects either "abc" or "def". The `"regexSubstitution"` key specifies the first returned match of the regular expression using "\\1". In this case "abc" is captured from the redirected URL and placed in the substitution.
+
+```json
+{
+  "id": 1,
+  "priority": 1,
+  "action": {
+    "type": "redirect",
+    "redirect": {
+      "regexSubstitution": "https://\\1.xyz.com/"
     }
   },
-  {
-    "id": 10,
-    "priority": 2,
-    "action": {
-      "type": "modifyHeaders",
-      "responseHeaders": [
-        { "header": "h1", "operation": "remove" },
-        { "header": "h2", "operation": "set", "value": "v2" },
-        { "header": "h3", "operation": "append", "value": "v3" }
-      ]
-    },
-    "condition": { "urlFilter": "headers.com/123", "resourceTypes": ["main_frame"] }
-  },
-  {
-    "id": 11,
-    "priority": 1,
-    "action": {
-      "type": "modifyHeaders",
-      "responseHeaders": [
-        { "header": "h1", "operation": "set", "value": "v4" },
-        { "header": "h2", "operation": "append", "value": "v5" },
-        { "header": "h3", "operation": "append", "value": "v6" }
-      ]
-    },
-    "condition": { "urlFilter": "headers.com/12345", "resourceTypes": ["main_frame"] }
+  "condition": {
+    "regexFilter": "^https://www\\.(abc|def)\\.xyz\\.com/",
+    "resourceTypes": [
+      "main_frame"
+    ]
+  }
+}
+```
+
+#### Headers {: #headers }
+
+The following example removes all cookies from both a main frame and any sub frames.
```

Comment:
```suggestion
The following example removes all cookies from a main frame and sub frames.
```

Reply:
What's your reasoning? Mine sounds more specific to my ears.